queue.go

```diff
@@ -18,24 +18,77 @@ package marathon
 
 import (
 	"fmt"
+	"time"
 )
 
-// Queue is the definition of marathon queue
 type Queue struct {
 	Items []Item `json:"queue"`
 }
 
-// Item is the definition of element in the queue
+// Queue is the definition of marathon queue
```

Comment:
Looks like you moved some comments

Reply:
Oh right I should adjust some of the comments.